configs/repvgg/metafile.yml

```diff
@@ -0,0 +1,219 @@
+Collections:
+  - Name: RepVGG
+    Metadata:
+      Training Data: ImageNet
+      Epochs: 200
+      Batch Size: 256
+      Architecture:
+        - re-parameterization Convolution
+        - VGG-style Neural Network
+    Paper:
+        URL: https://arxiv.org/abs/2101.03697
+        Title: 'RepVGG: Making VGG-style ConvNets Great Again'
+    README: configs/repvgg/README.md
+
+Models:
+  - Name: repvggA0_4xb64-coslr-120e_in-1k
+    In Collection: Repvgg
+    Config: configs/repvgg/repvggA0_4xb64-coslr-120e_in-1k.py
+    Metadata:
+      FLOPs: 1520000000
+      Parameters: 9110000
+      Training Data: ImageNet
```

Comment:
`Training Data` is shared by all models, just put it in the `RepVGG` collection. It's not necessary to put it in each model.